terraform/deployments/cluster-infrastructure/main.tf

```diff
@@ -34,13 +34,25 @@ locals {
       name_prefix = var.cluster_name
       # TODO: set iam_role_permissions_boundary
       # TODO: apply provider default_tags to instances; might need to set launch_template_tags.
-      desired_size               = var.workers_size_desired
-      max_size                   = var.workers_size_max
-      min_size                   = var.workers_size_min
-      instance_types             = var.workers_instance_types
+      desired_size   = var.workers_size_desired
+      max_size       = var.workers_size_max
+      min_size       = var.workers_size_min
+      instance_types = var.workers_instance_types
+      update_config  = { max_unavailable = 1 }
+      # TODO(#1201): remove disk_size and use_custom_launch_template after AL2023 rollout.
+      use_custom_launch_template = var.govuk_environment == "integration"
       disk_size                  = var.node_disk_size
-      use_custom_launch_template = var.govuk_environment == "integration" # TODO(#1201): remove with AL2023 rollout.
-      update_config              = { max_unavailable = 1 }
+      block_device_mappings = {
+        xvda = {
+          device_name = "/dev/xvda"
+          ebs = {
+            volume_size           = 60
```

Comment:
Do we want to set this to `var.node_disk_size` or shall we leave this hard-coded for now?

Reply:
Oh - also, should we add the same mapping to the ARM group?

Reply:
Whoops, good catches, thanks!